index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div id="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <div class="menu-items">
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div id="burger">
+        <img src="/images/icon-Menu.svg" alt="burger_icon" />
+      </div>
+    </nav>
+ <section class="headline_section">
+<h1 class="primary_text">I’m Emery.
```

Comment:
- [ ] I genuinely appreciate 🎉🎉your efforts in adding the section tag for the headline section. However, it appears that your design does not align with the suggested Figma design. It is essential to ensure that you follow the project requirements diligently.

- [ ] Please make sure that your website **precisely matches the design** in terms of elements such as _font_, _colors_, _images_, _text_, and _margins_. You can refer to [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) for guidance.